Homework_2/UserActions/FibonacciNumbersUserAction.cs

```diff
@@ -0,0 +1,75 @@
+namespace Homework_2.UserActions;
+
+public class FibonacciNumbersUserAction : IUserAction
+{
+    public void MakeAction()
+    {
+        while (true)
+        {
+            Console.ForegroundColor = ConsoleColor.Blue;
+            Console.Write("Введите число Фибоначчи: ");
+            Console.ForegroundColor = ConsoleColor.White;
+        
+            if (int.TryParse(Console.ReadLine(), out int fibbonachiNumber) && fibbonachiNumber >= 0)
+            {
+                int result = CalculateFibonachi(fibbonachiNumber);
+
+                Console.WriteLine($"Полученный результат: {result}");
+
+                while (true)
+                {
+                    Console.WriteLine("Желаете повторить вычисление чисел Фибоначчи? y/n");
+
+                    string lowerInputKey = Console.ReadLine().ToLower();
+
+                    if (lowerInputKey == "y")
+                    {
+                        break;
+                    }
+
+                    if (lowerInputKey == "n")
+                    {
+                        return;
+                    }
+                }
+            }
+            else
+            {
+                Console.ForegroundColor = ConsoleColor.Red;
+                    
+                while (true)
+                {
+                    Console.WriteLine("Введены некорректные данные, попробовать ещё раз? y/n");
+
+                    string lowerInputKey = Console.ReadLine().ToLower();
+
+                    if (lowerInputKey == "y")
+                    {
+                        break;
+                    }
+
+                    if (lowerInputKey == "n")
+                    {
+                        return;
+                    }
+                }
+            }
+        }
+    }
+
+    private int CalculateFibonachi(int number)
```

Comment:
there is only one usage of this function and it's not meant to be reused. Consider using comment to highlight calculation of Fibonacci numbers